assets/src/edit-story/components/form/radioGroup.js

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+import { rgba } from 'polished';
+
+const RadioButton = styled.div`
+  min-height: 30px;
+  margin: 10px 0;
+`;
+
+const Radio = styled.input`
+  opacity: 0;
+  position: absolute;
+  :focus + label {
```

Comment:
Should use:
```suggestion
  :focus + ${Label} {
```

To make sure it's updated if the label ever change class/element.